gradle.properties

```diff
@@ -9,7 +9,6 @@ pluginVersion = 0.1.3
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
 pluginSinceBuild = 213
-pluginUntilBuild = 999.*
```

Comment:
There are warnings in the build that this is invalid - they recommend leaving it off entirely to allow it to be used with all future builds.

Reply:
https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html